index.js

```diff
@@ -0,0 +1,306 @@
+                                <li class="P3" class="back-dev">${item.year}</li>
+                            </ul>
+                            <p>
+                                ${item.description}
+                              </p>
+                              <ul class="T1">
+                                <li>${item.skills.skill1}</li>
+                                <li>${item.skills.skill2}</li>
+                                <li>${item.skills.skill3}</li>
+                              </ul>
+                              <button class="see-mobile">See Project</button>
+                        </div>
+                    </div>
+
+                </div>
+  </div>
+    `;
+  const mobileSectionCont = document.querySelector('.main-container');
+  mobileSectionCont.appendChild(forMobile);
+  const mobilePop = document.querySelectorAll('.see-mobile');
+
+  mobilePop.forEach((pop) => {
+    pop.addEventListener('click', (e) => {
+      let { id } = e.target;
+      id = Number(id);
+      if (id === item.id) {
+        const itemMobileDiv = document.createElement('div');
+        itemMobileDiv.className = 'new-items';
+        itemMobileDiv.innerHTML = `
+          <h1>${item.name}</h1>
+          <div class="canopy-div-pop">
+            <p>${item.title}</p>
+            <img src="${item.circle}" alt="" />
+            <p>${item.stack}</p>
+            <img src="${item.circle}" alt="" />
+            <p>${item.year}</p>
+          </div>
+          <img src="${item.mobileImg}" class="pop-desk-img" alt="Mobile Image">
+          <div class="description-cont">
+          <div class="description-p">
+            <p class="writeup-div">${item.description}</p>
+          </div>
+            <div class="description-btn-cont">
+             <button class="desktop-btn">${item.skills.skill1}</button>
+             <button class="desktop-btn">${item.skills.skill2}</button>
+             <button class="desktop-btn">${item.skills.skill3}</button>
+             <hr>
+             <div class="description-see">
+               <button class="see-project">See Live</button>
+               <button class="see-project">See Source</button>
+             </div>
+            </div>
+          </div>
+
+            `;
+
+        const backOverlay = document.querySelector('.mobile-backdrop1');
+        backOverlay.classList.add('modal-backdrop-show');
+
+        const modalContainer = document.querySelector('.modal-inner');
+
+        modalContainer.appendChild(itemMobileDiv);
+        modalContainer.classList.add('modal-inner-show');
+        // const cancelButton = document.querySelector("#modal-close");
+
+        // cancelButton.addEventListener("click", (e) => {
+        //     e.preventDefault();
+        //     itemMobileDiv.innerHTML = "";
+        //     modalContain.classList.remove("new-modal-show");
+        //     backOverlay.classList.remove("modal-backdrop-show");
+        // });
```

Comment:
[Optional] 

- [ ] Kindly remove all commented line of unused code ,commented line of code can couse a distraction from reading a code it make a code look messy,kindly make it clean  to make your easy to read.  
 can make your code